js/modules/handleBurgerButtons.js

```diff
@@ -0,0 +1,19 @@
+const menuOverlay = document.body.querySelector(".menu_overlay");
+const burgerBtn = document.body.querySelector(".menu_burger");
+const menuCloseBtn = menuOverlay.querySelector(".menu_close_btn");
+
+burgerBtn.addEventListener("click", openMenu);
+
+function openMenu(){
+	menuOverlay.classList.add("active");
+	document.body.classList.add("stopPageScroll");
+	document.addEventListener("click", closeMenu);
+}
+
+function closeMenu(event){
```

Comment:
there are must be 2 functions: open and close. There must not be any conditions inside. Close must only close menu, the same with open.
You can create the third function, something like this
```js
function closeByOverlay(){
   if(event.target === menuOverlay) closeMenu();
}
```

and set this function like a handler on click by overlay

Reply:
Done.